devops/actions/llvm_test_suite/action.yml

```diff
@@ -32,6 +32,7 @@ inputs:
 post-if: false
 runs:
   using: "composite"
+  timeout-minutes: 60
```

Comment:
Do you know why isn't `export LIT_OPTS="-v --no-progress-bar --show-unsupported --max-time 3600` enough (line 85)? Do we have timeouts in steps other than the LIT itself?

Reply:
I'm trying to reduce the impact of the issue we recently experienced with self-hosted runners. Our runners received commands, but didn't send any information back (i.e. logs and execution status). In this case, GHA severs waits for 6 hours, sends a signal to cancel the job and enqueues a task (which also looks like it hangs for 6 hours with no logs).
Here I'm trying to limit the wait from 6 to just 1 hour.